### ISSUE TITLE
argument -label is not recognize 

### ISSUE THREAD

**Issue by bananemure** [pommedeterresautee/fastrtext#20]
Hi @pommedeterresautee 
I run a supervised model using my own label string but it seems that we cannot set our own label since the option -label is not settable.

Unknown argument: -label

**Comment by pommedeterresautee** [comment]
Strange, I just tried and it works.
Can you post the command you use?

**Comment by bananemure** [comment]
This is the command...

pretrainModel <- '../embedvectors/allvec/allvec200.vec'
fastrtext::execute(commands = c("supervised", 
                                "-input", 'train_icd.tsv', 
                                "-output", 'icd_model', 
                                "-dim", 200, 
                                "-lr", 1, 
                                "-minn",4,
                                "-maxn",6,
                                "-loss", 'softmax',
                                "-label ","class__",
                                "-epoch", 20, 
                                "-wordNgrams", 3, 
                                "-verbose", 2,
                                "-thread",100,
                                "-pretrainedVectors" ,pretrainModel
                                
                                )
                   )
thanks

**Comment by pommedeterresautee** [comment]
you put a space in "-label " (after label)

**Comment by bananemure** [comment]
Ohhh sorry you are so right...
Thank again....
Great work with the package. A great + for the R users
